src/Mod/Test/TestApp.py

```diff
@@ -68,7 +68,13 @@ def All():
 def TestText(s):
     s = unittest.defaultTestLoader.loadTestsFromName(s)
     r = unittest.TextTestRunner(stream=sys.stdout, verbosity=2)
-    return r.run(s)
+    retval = r.run(s)
+    # Flushing to make sure the stream is written to the console
+    # before the wrapping process stops executing. Without this line 
+    # executing the tests from command line did not show stats
+    # and proper tarceback in some cases.

```

Comment:
In Windows; this seems to affect the Windows console, not the Linux terminal in general.